### DISCUSSION TITLE
dwg文件被错误识别为媒体文件

### DISCUSSION THREAD

**Discussion by lzy4ever** [localsend/localsend#558] [L-简体中文]
1.10版本iOS和安卓端开启自动保存到相册功能后无法接受cad图纸文件（后缀为.dwg格式），iOS端卡进度条，安卓端提示错误，关掉保存到相册功能后可以正常接收dwg文件，猜测把dwg文件错误识别为媒体文件后因无法保存到相册内导致传输失败，希望下一版本修复这个错误，谢谢

**Comment by vlate** [comment]
遇到相同问题。
1.17版本依然有这个问题。

确认，手机端取消“保存到相册”可避免这个问题。
